sdk/couchbase-core/src/options/agent.rs

```diff
@@ -262,3 +292,38 @@ impl Default for HttpConfig {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq)]
+#[non_exhaustive]
+pub struct OrphanReporterConfig {
```

Comment:
This should probably be in its own file as it doesn't pertain to the agent directly.